inc/functions-home.php

```diff
@@ -44,48 +44,25 @@ function limit_words( $words, $number = 14 ) {
  * @param string $title
  * @param string $description
  * @param string $image
+ * @param string $date
  * @return string
  */
-function display_card( $id, $url, $title, $description, $image ) {
+function display_card( $id, $url, $title, $description, $image, $date ) {
 
-	if ( $url ) {
+	$type = content_type( $url );
 
-		$og_data = get_meta_og_data( $url );
+	if ( !$fp = curl_init($url) ) {
```

Comment:
Hi @domingobishop - I'm a little confused by this condition and the role of `$fp`. Could you take me through it?